andy/src/main/java/nl/tudelft/cse1110/andy/execution/step/SourceCodeSecurityCheckStep.java

```diff
@@ -38,46 +38,37 @@ public void execute(Context ctx, ResultBuilder result) {
             return;
         }
 
-        if (!checkPackageName(code, result)) return;
-
         if (!checkForKeywords(code, result)) return;
     }
 
-    private boolean checkPackageName(String code, ResultBuilder result) {
-        Pattern pattern = Pattern.compile("^\\s*package\\s+delft\\s*;.*", Pattern.DOTALL);
-        if (!pattern.matcher(code).find()) {
-            result.compilationSecurityFail("The package name of your solution must be \"delft\"");
-            return false;
-        }
-
-        return true;
-    }
-
     private boolean checkForKeywords(String code, ResultBuilder result) {
-        String reflectionMsg = "Using reflection in your code is not allowed";
-        String seleniumUnsupportedDriverMsg = "Please use HtmlUnitDriver as Andy does not have a browser installed";
+        String msg = "It is not allowed to use '%s' in your code";
+        String seleniumUnsupportedDriverMsg = "Please use CSE1110.createDriver() to instantiate a Selenium driver";
```

Comment:
This message seems rather specific. Maybe we can change it to: "Please use the provided method to instantiate a Selenium driver".

Reply:
Sure, I changed it